nx-libs.spec

```diff
@@ -804,23 +771,6 @@ ln -s -f ../../../../%{_lib}/libXext.so.6 %{buildroot}%{_libdir}/nx/X11/Xinerama
 %defattr(-,root,root)
 %{_libdir}/libNX_Xau.so.6*
 
-%files -n libNX_Xcomposite1
-%defattr(-,root,root)
-%{_libdir}/libNX_Xcomposite.so.1*
-
-%files -n libNX_Xdamage1
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdamage.so.1*
-
-%files -n libNX_Xdmcp-devel
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdmcp.so
-%{_includedir}/nx/X11/Xdmcp.h
-
-%files -n libNX_Xdmcp6
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdmcp.so.6*
-
```

Comment:
rpmlint will accept empty packages. But a %files section **must** exist per package, or otherwise no RPM will be created at all.

The %files section can be empty.

This said, the whole dummy package stuff isn't a good idea and severely frowned upon by SUSE staff. I'll have to get a clearer cross-distro understanding here.